LedRGB.cpp

```diff
@@ -0,0 +1,38 @@
+#include "LedRGB.h"
+
+LedRGB::LedRGB() : REDPIN(0), GREENPIN(0), BLUEPIN(0) {
+}
+
+LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
+    : REDPIN(redPin), GREENPIN(greenPin), BLUEPIN(bluePin) {
+    //	pinMode(redPin, OUTPUT);
```

Comment:
an rgb led is basically 3 regular leds so lets reuse the code you already wrote. (use 3 led objects).

Refactor the rest of the functions to use leds instead